R-package/demo/caret_wrapper.R

```diff
@@ -1,5 +1,4 @@
 # install development version of caret library that contains xgboost models
-devtools::install_github("topepo/caret/pkg/caret")
```

Comment:
XGBoost support has been in `{caret}` since at least 2015 (https://github.com/topepo/caret/commit/017c7cb75f9224a012c9cdaf819d0e11f0af4ec9). I think it's safe to remove this from this demo and assume that anyone with `{caret}` installed has a version with XGBoost support.